src/main/java/studentdirectory/controller/FileController.java

```diff
@@ -0,0 +1,76 @@
+package studentdirectory.controller;
+
+import static studentdirectory.constants.Path.FILE_PATH;
```

Comment:
should we import as static or should we use full name.

Reply:
https://docs.oracle.com/javase/7/docs/technotes/guides/language/static-import.html - It is mentioned here that we need to use static imports only when we know that the variable declared are going to be used at more than 1 places and in that cases using static import will improve our readability as compared to writing fully-qualified names